modules/serverless-service-mesh-resources.adoc

```diff
@@ -122,14 +111,12 @@ spec:
     kind: Service
     name: istio-ingressgateway
 ----
-
 <1> The {product-title} route must be created in the same namespace as the
 ServiceMeshControlPlane. In this example, the ServiceMeshControlPlane is
```

Comment:
Should these `ServiceMeshControlPlane` instances be marked up?

Reply:
Good call, checked in the OSSM docs and they use "`ServiceMeshControlPlane` resource" now so I changed it to that.